### PR TITLE
docs: align automated review categories

### LINKED ISSUE #2104: Drift: root CLAUDE.md advertises 13 review categories including "Agent skill freshness" — source-of-truth review-patterns.md has 11 and no such category
## What drifted

Root [`CLAUDE.md`](../blob/main/CLAUDE.md) (lines 115–133) and [`docs/root-details.md`](../blob/main/docs/root-details.md) (lines 31–55) advertise **13 numbered PR-review categories**, one of which is **#10 \"Agent skill freshness\"**. They both point to [`docs/review-patterns.md`](../blob/main/docs/review-patterns.md) as the \"full catalog\" / **\"single source of truth\"** for the automated reviewers.

But `docs/review-patterns.md` (heading `## Detection categories` at line 56, then `### 1.` through `### 11.` down to line 337) has only **11 numbered categories**, uses **different numbering** past #9, and does **not contain \"Agent skill freshness\" anywhere** — not as `#10`, not under another number, not as an unnumbered aside.

For extra confusion, [`docs/CLAUDE.md:76`](../blob/main/docs/CLAUDE.md#L76) still describes `review-patterns.md` as covering **\"the nine recurring blind-spot categories\"** — a number that was accurate several categories ago.

## Evidence — the three lists side by side

| # in root `CLAUDE.md` / `root-details.md` | `docs/review-patterns.md` (source of truth) |
| --- | --- |
| 1. Error-path coverage | 1. Error-path coverage gaps |
| 2. UI state preservation | 2. UI state preservation |
| 3. Cross-runtime parity | 3. Cross-runtime consistency |
| 4. CDP edge cases | 4. CDP / Chrome integration edge cases |
| 5. Native/macOS permissions | 5. macOS / native permissions |
| 6. Model metadata / provider pipeline | 6. Model metadata / provider pipeline gaps |
| 7. Test coverage | 7. Test-coverage blind spots |
| 8. Follower wiring parity | 8. Follower surface wiring parity |
| 9. Origin/bridge routing | 9. Origin / bridge routing contract |
| **10. Agent skill freshness** | **10. Layer-stack import direction (back-edges)** |
| 11. Transcript export | 11. Untyped string-keyed bags (`Record<string, unknown>`) |
| 12. Layer import direction | *(Transcript export — unnumbered `##` section at line 341)* |
| 13. Untyped string-keyed bags | — |

Specific citations (claim vs. code):

- Claim: `CLAUDE.md:128` — `10. Agent skill freshness`.
- Claim: `docs/root-details.md:48-49` — `10. **Agent skill freshness** — shell command changes → update matching vfs-root/workspace/skills/*/SKILL.md.`
- Code: `docs/review-patterns.md:56` (`## Detection categories`) then `### 1.`–`### 11.` at lines 58, 75, 92, 109, 126, 145, 177, 195, 244, 271, 309 — no \"Agent skill freshness\" heading exists (grep confirms zero hits for the phrase in `docs/review-patterns.md`).
- Code: `docs/review-patterns.md:341` — Transcript export appears as an **unnumbered** `## Transcript export — redaction boundary and protocol parity` section, not `#11` as the root claims.
- Claim: `docs/CLAUDE.md:76` — `the nine recurring blind-spot categories` (actual count in review-patterns.md is eleven numbered + one unnumbered).

## Why this is Drift (and impactful)

- `docs/review-patterns.md` is **explicitly declared** the \"single source of truth\" (lines 3–4) that all three automated reviewers (Claude action, Codex via root `CLAUDE.md`, Copilot via `.github/copilot-instructions.md`) feed from.
- Root `CLAUDE.md:133` even says: *\"When you change a category, update \`docs/review-patterns.md\` (source of truth)\"* — but the root itself is out of sync with that source of truth.
- The root advertises a category (**\"Agent skill freshness\"**) that the source of truth does not carry, so a reviewer told to check \"category #10\" gets a different rule depending on which file it reads (Agent skill freshness vs. Layer-stack import direction).
- Numbers `#11`/`#12`/`#13` in the root map to **different** categories in the source of truth (Transcript export vs. Layer stack, Layer stack vs. Untyped bags, Untyped bags vs. nothing).
- The `docs/CLAUDE.md` \"nine\" phrasing reinforces the stale mental model for anyone reading the tier map first.

Category numbering is the *interface* between the condensed checklist and the source of truth; when the two lists disagree, the reviewers disagree. That's the sin: the descriptions in `CLAUDE.md` files are trusted over what `review-patterns.md` actually says.

## Suggested remediation

Pick one and reconcile the affected files (`CLAUDE.md`, `docs/root-details.md`, `docs/review-patterns.md`, `docs/CLAUDE.md`, plus `.github/copilot-instructions.md`) in the same PR:

1. **Add \"Agent skill freshness\" to `docs/review-patterns.md`** as its own numbered `### N. Agent skill freshness` section (trigger patterns, precedent — see `docs/playwright-cli-sync.md` — and remediation). Promote the Transcript export section to a numbered category (matching root's #11). Then push identical numbers into root `CLAUDE.md`, `docs/root-details.md`, and `.github/copilot-instructions.md`.
2. **Or remove #10 \"Agent skill freshness\"** from root `CLAUDE.md` and `docs/root-details.md` if that category is no longer meant to be reviewer-enforced (the `adding-slicc-features` skill and `docs/playwright-cli-sync.md` already document the workflow), and shift the tail down.

Either way: fix `docs/CLAUDE.md:76` — replace `the nine recurring blind-spot categories` with a phrasing that will not drift (e.g. *\"the numbered blind-spot categories\"*), since the literal count breaks every time a category is added.

Occurrence/scope: four documentation files affected; touched every time a reviewer or human reads the checklist.

### PR DESCRIPTION
**Pull request by trieloff**
Closes #2104

## Solution
Makes the 13-category source catalog and every condensed reviewer checklist use one numbering scheme, including first-class skill-freshness and transcript-export categories.

## Testing
- Documentation, lint, touched-debt, typecheck, browser builds, and bundle budgets pass.
- Full tests: 14,239 pass; one pre-existing fake-LLM sequential-request ordering test fails consistently. Aggregate build additionally lacks the Swift toolchain.

---
Pull Request opened by [Augment Code](https://www.augmentcode.com/) | [View session](https://cosmos.augmentcode.com/session?agentId=01M07G98V9JCDG7AC9DJN315R7&panel=chat)

### CODE REVIEW

**Comment by trieloff** [comment]
<sup>[**PR Author Agent**](https://cosmos.augmentcode.com/session?agentId=01M07G98V9JCDG7AC9DJN315R7)⚡</sup>

👋 I've got this PR.
I'll handle review feedback, CI failures, and merge conflicts. Marking it ready and picking reviewers are your call.

File: docs/review-patterns.md [hunk 1]
```diff
@@ -338,7 +338,28 @@ occurrence not in the frozen baseline; the baseline is a one-way ratchet — shr
...
+  procedure and its `.claude/skills/` discovery alias.
+
+**Historical precedent** — **PR #1130** aligned SLICC's `playwright-cli` with the official
+CLI, closing 39 command gaps and refreshing the runtime skill with the newly supported
+commands. Its sync script detects command-surface drift; the skill update keeps that surface
+usable by agents after it lands.
+
+**Remediation** — follow the cross-reference checklist in
+[adding-slicc-features](../.agents/skills/adding-slicc-features/SKILL.md): update the matching
+runtime and developer skills in the same PR, run `npm run lint:skill-router`, and run the
```
[the remark this answers is not in the document]
Comment (trieloff) [reply docs/review-patterns.md]: <sup>[**PR Author Agent**](https://cosmos.augmentcode.com/session?agentId=01M07G98V9JCDG7AC9DJN315R7)⚡</sup>

Fixed in 1159e65ef: remediation now scopes runtime skills to agent-facing capabilities and developer skills to developer workflows.
